.github/workflows/build-and-test.yml

```diff
@@ -12,10 +12,10 @@ on:
       - dev
```

Comment:
lets also remove `dev`

Reply:
There are a lot of issues with the workflow files, I'll do a second PR